### ISSUE TITLE
Latest version (5.0.2) throws errors.

### ISSUE THREAD

**Issue by asha23** [JeffreyWay/laravel-mix#2349]
Just thought I'd flag.

I have rolled back to (v5.0.1), as Mix has stopped working on the latest version. (v5.0.2)

I'm running the latest versions of node and npm.

Throws the following error:

```
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn node_modules/webpack/bin/webpack.js ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    [... lines matching original stack trace ...]
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dt-***@1.2.0 development: `cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dt-***r@1.2.0 development script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/***/.npm/_logs/2020-03-16T16_43_07_498Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dt-***@1.2.0 dev: `npm run development`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dt-***@1.2.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/***/.npm/_logs/2020-03-16T16_43_07_512Z-debug.log
```

I tried all of the usual methods, when this kind of error occurs. IE.

```
rm -rf node_modules
rm package-lock.json yarn.lock
npm cache clear --force
npm install
```
Nothing seemed to fix it. Rolling back worked fine though.

I'm using it standalone, not as part of a Laravel build.

My deps are

```
    "browser-sync": "^2.26.7",
    "browser-sync-webpack-plugin": "^2.2.2",
    "copy-webpack-plugin": "^5.1.1",
    "cross-env": "^7.0.2",
    "fontawesome": "^5.6.3",
    "imagemin-webpack-plugin": "^2.4.2",
    "laravel-mix": "5.0.1",
    "node-sass": "^4.13.1",
    "photo-sphere-viewer": "^3.6.0",
    "resolve-url-loader": "^3.1.1",
    "sass": "^1.26.3",
    "sass-loader": "^8.0.2",
    "vue": "^2.6.11",
    "vue-template-compiler": "^2.6.11"
    "@fortawesome/fontawesome-free": "^5.12.1",
    "laravel-mix-imagemin": "^1.0.3",
    "three": "^0.114.0",
    "uevent": "^2.0.0"
```

**Comment by indigoswann** [comment]
Same issue, totally blew away my `node_modules` tried cleaning npm cache, rm -rfing all, but no dice.

**Comment by Krisell** [comment]
There a 5.0.4 now which works, so just run `npm update`.

**Comment by laracasts** [comment]
Looks like it was a strange shrink-wrapping issue. All fixed now in 5.0.4.

**Comment by slainr** [comment]
I was facing the same issue on laravel-mix 5.0.2, the only way to solve the errors was upgrading to 5.0.4!
